images/installer/Dockerfile.upi.ci

```diff
@@ -8,8 +8,10 @@ WORKDIR /go/src/github.com/openshift/installer
 COPY . .
 RUN hack/build.sh
 
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:cli as cli
```

Comment:
this change already merged https://github.com/openshift/installer/pull/1497 why am i still seeing this?

Reply:
I didn't rebase yet, sorry.

This PR is just for CI testing, so that all fixes would be in a single PR. I'll be cherrypicking changes from here to separate PRs if required